packages/python-packages/api-stub-generator/setup.cfg

```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
+universal=1
```

Comment:
We call this universal right? But we use `enum` without a dep on `enum34` 

By def I think this is a `py3` only package. Maybe put a `python_requires` of `>= 3.4`?

I do NOT think it's worth to make this a fully universal package. That's just my opinion though.